db.py

```diff
@@ -15,10 +15,12 @@ def start():
     c.execute("CREATE TABLE IF NOT EXISTS message_counts (user TEXT, count INTEGER)")
     c.execute("CREATE UNIQUE INDEX IF NOT EXISTS message_counts_idx ON message_counts (user)")
     c.execute("CREATE TABLE IF NOT EXISTS message_hashes (user TEXT, message_id TEXT, message_hash TEXT, time_sent TEXT)")
-    c.execute("CREATE TABLE IF NOT EXISTS image_hashes (hash TEXT, message_id TEXT, channel_id TEXT, guild_id TEXT)")
```

Comment:
Why did we move this? Does this provide optimal performance?

Reply:
index creation for the other tables happens on the immediately following line, so this was to be more consistent.

Reply:
I'm thinking we refine this to fetch the past month (by default, parameterised). Then when we build the graph, default all empty days to a 0/None rating -- I would hope None would leave a gap in the graph, though a zero would encourage more frequent posting